jinahub/indexers/query/vector/AnnoyIndexer/Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM jinaai/jina:1.1.10
+FROM jinaai/jina:master
+
+# install and upgrade pip
+RUN apt-get update && apt-get install -y python3.7 python3.7-dev python3-pip git
+RUN python3.7 -m pip install --upgrade pip
 
 # setup the workspace
-COPY . /workspace
-WORKDIR /workspace
+COPY requirements.txt /requirements.txt
```

Comment:
Remove this line and move the COPY and WORKDIR statement before the RUN pip

Reply:
fixed